spring-rabbit/src/test/java/org/springframework/amqp/rabbit/annotation/RabbitListenerAnnotationBeanPostProcessorTests.java

```diff
@@ -269,6 +304,16 @@ public void handleIt(String body) {
 		}
 	}
 
+	@Component
+	static class MultipleBindingsTestBean {
+
+		@RabbitListener(bindings = {
+				@QueueBinding(exchange = @Exchange("my_exchange"), value = @org.springframework.amqp.rabbit.annotation.Queue(value = "my_queue", arguments = @Argument(name = "foo", value = "bar")), key = "red"),
+				@QueueBinding(exchange = @Exchange("my_exchange"), value = @org.springframework.amqp.rabbit.annotation.Queue("my_queue"), key = "yellow") })
```

Comment:
Wouldn't it be a lot easier to change the key property to be an array...

	String[] key() default {};

... and then

		@RabbitListener(bindings = {
				@QueueBinding(exchange = @Exchange("my_exchange"), 
						value = @org.springframework.amqp.rabbit.annotation.Queue("my_queue"), 
							key = { "red", "yellow" }) })
		public void handleIt(String body) {
		}

?